features/autoconsent.json

```
@@ -562,6 +562,10 @@
         {
             "domain": "stiga.com",
             "reason": "https://github.com/duckduckgo/privacy-configuration/pull/3109"
+        },
+        {
+            "domain": "service.uk.gov",
```

Sorry @jdorweiler, I think the domain is wrong:
```suggestion
            "domain": “service.gov.uk",
```

---

ah thanks